huxley/core/admin/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 from .assignment import AssignmentAdmin
 from .committee import CommitteeAdmin
+from .committee_feedback import CommitteeFeedbackAdmin
```

Comment:
Remember to get rid of this line

Reply:
Done

Reply:
That's what made Travis fail this time whoops
![screenshot from 2018-01-02 11-49-12](https://user-images.githubusercontent.com/22579850/34497632-3086487e-efb3-11e7-9563-c364dc2a6e51.png)